index.js

```diff
@@ -110,14 +110,24 @@ function createLoaderRules(languages, features, workers, outputPath, publicPath)
     workerPaths['razor'] = workerPaths['html'];
   }
 
+  // Determine the public path from which to load worker JS files. In order of precedence:
+  // 1. Plugin-specific public path.
+  // 2. Dynamic runtime public path.
+  // 3. Compilation public path.
+  const pathPrefix = Boolean(pluginPublicPath)
+    ? JSON.stringify(pluginPublicPath)
+    : `typeof __webpack_public_path__ === 'string' ` +
+      `? __webpack_public_path__ ` +
```

Comment:
Including the fix from #63 here since I'm touching the same code – this should be referenced as a free variable and not off of window as documented [here](https://webpack.js.org/guides/public-path/#on-the-fly).